data_steward/gcloud/bq/__init__.py

```diff
@@ -73,7 +73,12 @@ def get_table_schema(self, table_name: str, fields=None) -> list:
         if fields:
             fields = fields
         else:
-            fields = fields_for(table_name)
+            if table_name.endswith('identity_match'):
+                fields = fields_for('identity_match')
+            elif table_name.endswith('participant_match'):
+                fields = fields_for('participant_match')
```

Comment:
I know this is directly from the patch file I gave you, but I was using the quick and dirty method for the patch.  These strings should be replaced by values defined in common.py.